### PR TITLE
update: doc update and new script section

### PR DESCRIPTION
**Pull request by robin-da**
updated logo, some parts of the docs and added a section on daml script.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Review by nemanja-da — APPROVED** [review]
ty

File: _config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-title: The Fast Track to DAML
-baseurl: "."
-description: "The Fast Track to DAML - DAML Cheat Sheet"
+title: The Fast Track to Daml
+baseurl: ""
+description: "The Fast Track to Daml - DAML Cheat Sheet"
```
Comment (anthonylusardi-da) [inline _config.yml:3]:
```suggestion
description: "The Fast Track to Daml - Daml Cheat Sheet"
```

File: _includes/en/Scripts.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  alice <- allocateParty "Alice" 
+  bob <- allocateParty "Bob"
+  c <- submit alice $ createCmd NewContract with ...
+  submit bob $ exerciseCmd c Accept with ...
+```
+
+Scripts are compiled like usual Daml code to a `dar` package with the `daml build` command.
+
+| -------------------- | ----------------------------------------------------- |
+| Running a script | `daml script --dar example-0.0.1.dar --script-name ScriptExample:test --ledger-host localhost --ledger-port 6865`
```
Comment (anthonylusardi-da) [inline _includes/en/Scripts.md:19]: Could this be `ModuleName:scriptFunction` so it's more clear to people how to refer to their scripts?

Comment (anthonylusardi-da) [inline _includes/en/Scripts.md:19]:
```suggestion
| Running a script | `daml script --dar example-0.0.1.dar --script-name ModuleName:scriptFunction --ledger-host localhost --ledger-port 6865`
```

File: _includes/en/Scripts.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+| Running a script with initial arguments given | `daml script --dar example-0.0.1.dar --input-file arguments_in_damllf_json.json --script-name ScriptExample:test --ledger-host localhost --ledger-port 6865` |
```
Comment (anthonylusardi-da) [inline _includes/en/Scripts.md:20]:
```suggestion
| Running a script with initial arguments given | `daml script --dar example-0.0.1.dar --input-file arguments_in_damllf_json.json --script-name ModuleName:scriptFunction --ledger-host localhost --ledger-port 6865` |
```

Comment (anthonylusardi-da) [inline _includes/en/Scripts.md:20]: Also perhaps a link to a reference on how to produce `arguments_in_damllf_json.json`?

**Review by anthonylusardi-da — COMMENTED** [review]
Done, looks great, just a few comments/small changes

File: _includes/en/Scripts.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+| Allocating a party on the ledger | `alice <- allocateParty "Alice"` |
+| List all known parties on the ledger | `parties <- listKnownParties` |
+| Query for a given contract template visible to a given party | `query @ExampleTemplate alice` |
+| Create a new contract | `createCmd ExampleTemplate with ...` |
+| Exercise a choice on a contract | `exerciseCmd contractId ChoiceName with ... ` |
+| Exercise a choice on a contract by contract key | `exerciseByKeyCmd contractKey ChoiceName with ... ` |
+| Create and then exercise a choice on the created contract | `createAndExerciseCmd (ExampleTemplate with ... ) (ChoiceName with ...)` |
+| Pass time on the ledger (only applicable for a ledger running in **STATIC TIME MODE**) |  `passTime (hours 10)` |
```
Comment (cocreature) [inline _includes/en/Scripts.md:28]: maybe mention explicitly that this includes daml studio and daml thereby also daml test.

**Review by cocreature — COMMENTED** [review]
Nice!

**Review by cocreature — APPROVED** [review]
and now with the correct button

File: _includes/en/Scripts.md (hunk 2 above)
Comment (robin-da) [reply _includes/en/Scripts.md]: committed your suggestions.

File: _includes/en/Scripts.md (hunk 4 above)
Comment (robin-da) [reply _includes/en/Scripts.md:28]: added a note.

File: _includes/en/Scripts.md (hunk 3 above)
Comment (robin-da) [reply _includes/en/Scripts.md]: That would just point to the daml scripts docs, that's already mentionened.

**Pull request merged**